src/utils/vc.js

```diff
@@ -114,17 +174,25 @@ export async function checkRevocationStatus(credential, revocationApi) {
   if (!revocationApi.dock) {
     throw new Error('Only Dock revocation support is present as of now.');
   } else {
-    if (!hasDockRevocation(credential)) {
+    const statuses = await getCredentialStatuses(credential);
+
+    if (!hasDockRevocation(statuses)) {
```

Comment:
the old method did, but now this supports multiple statuses. see the function body. getCredentialStatuses returns an array

Reply:
`hasDockRevocation` is the only place that checks:

```js
jsonld.getValues(status, '@type').includes(RevRegType) && id.startsWith(DockRevRegQualifier)
```

We should be checking that every for every status. It's not enough that just one status satisfies that check because we end up doing a revocation lookup for every status in the list.

Consider this credential (look at credentialStatus):

```js
{
  '@context': 'https://www.w3.org/2018/credentials/v1',
  id: 'uuid:0x9b561796d3450eb2673fed26dd9c07192390177ad93e0835bc7a5fbb705d52bc',
  type: [
    'VerifiableCredential',
    'AlumniCredential',
  ],
  issuanceDate: '2020-03-18T19:23:24Z',
  credentialSubject: {
    id: 'did:dock:5GL3xbkr3vfs4qJ94YUHwpVVsPSSAyvJcafHz1wNb5zrSPGi',
    emailAddress: 'johnsmith@example.com',
    alumniOf: 'Example University',
  },
  credentialStatus: [
    {
      id: 'rev-reg:dock:0xf16baed331323c32765faafb7ff15e2f6c07f9cf3a0f96578125e85c71be7574',
      type: 'CredentialStatusList2017',
    },
    {
      id: 'https://revocations.com/registry12345',
      type: 'CredentialStatusList2017',
    },
  ],
  issuer: 'did:dock:5GUBvwnV6UyRWZ7wjsBptSquiSHGr9dXAy8dZYUR9WdjmLUr',
  proof: {
    type: 'Ed25519Signature2018',
    created: '2020-04-22T07:50:13Z',
    jws: 'eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..GBqyaiTMhVt4R5P2bMGcLNJPWEUq7WmGHG7Wc6mKBo9k3vSo7v7sRKwqS8-m0og_ANKcb5m-_YdXC2KMnZwLBg',
    proofPurpose: 'assertionMethod',
    verificationMethod: 'did:dock:5GUBvwnV6UyRWZ7wjsBptSquiSHGr9dXAy8dZYUR9WdjmLUr#keys-1',
  },
}
```